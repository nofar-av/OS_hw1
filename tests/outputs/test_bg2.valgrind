==26869== Memcheck, a memory error detector
==26869== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==26869== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==26869== Command: /home/omer/OS_wet1/smash
==26869== Parent PID: 26865
==26869== 
==26869== 
==26869== FILE DESCRIPTORS: 4 open at exit.
==26869== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_bg2.valgrind
==26869==    <inherited from parent>
==26869== 
==26869== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_bg2.err
==26869==    <inherited from parent>
==26869== 
==26869== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_bg2.out
==26869==    <inherited from parent>
==26869== 
==26869== Open file descriptor 0:
==26869==    <inherited from parent>
==26869== 
==26869== 
==26869== HEAP SUMMARY:
==26869==     in use at exit: 0 bytes in 0 blocks
==26869==   total heap usage: 116 allocs, 116 frees, 87,313 bytes allocated
==26869== 
==26869== All heap blocks were freed -- no leaks are possible
==26869== 
==26869== For counts of detected and suppressed errors, rerun with: -v
==26869== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
