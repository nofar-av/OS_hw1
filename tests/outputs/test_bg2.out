smash> submitted by Eran
smash> smash> smash: got ctrl-Z
smash: process 5302 was stopped
smash> smash> [1] sleep 100 : 5302 0 secs (stopped)
[2] sleep 200& : 5359 0 secs
smash> smash> smash> smash> sleep 100 : 5302
smash> [1] sleep 100 : 5302 0 secs
[2] sleep 200& : 5359 0 secs
smash> smash: got ctrl-Z
smash: process 5360 was stopped
smash> smash: got ctrl-Z
smash: process 5404 was stopped
smash> smash> sleep 400 : 5404
smash> [1] sleep 100 : 5302 1 secs
[2] sleep 200& : 5359 1 secs
[3] sleep 300 : 5360 1 secs (stopped)
[4] sleep 400 : 5404 0 secs
[5] sleep 500& : 5455 0 secs
smash> smash: sending SIGKILL signal to 5 jobs:
5302: sleep 100
5359: sleep 200&
5360: sleep 300
5404: sleep 400
5455: sleep 500&
